mobbage/__init__.py

```diff
@@ -307,15 +308,15 @@ def put(self, job_dict, args):
         if "headers" in job:
             if not isinstance(job.headers, list):
                 raise Exception("Headers must be in list form")
-            header_list = job.headers + args.header
+            header_list = job.headers + args.headers
         else:
-            header_list = args.header
+            header_list = args.headers
 
         # Convert our list of colon-delimited k:v pairs to a dict
         header_dict = {}
         for kv in header_list:
             try:
-                key, val = map(strip, ':'.split(kv))
+                key, val = map(string.strip, kv.split(':'))
```

Comment:
👍 You could alternatively do the following and avoid the `import` at the top, but I'm fine with what you did here.
```
key, val = [s.strip() for s in kv.split(':')]
```